packages/website/src/common/NavBar/index.tsx

```diff
@@ -115,7 +116,22 @@ const NavBar = ({ searchLocation, classes }: NavBarProps) => {
                       open={Boolean(anchorEl)}
                       onClose={handleMenuClose}
                     >
+                      {user.administeredReefs?.map((reef, index) => {
```

Comment:
Maybe worth destructuring reef
```suggestion
                      {user.administeredReefs?.map(({ name, region, id }, index) => {
```